src/shared/ui/date-lang/date-lang.tsx

```diff
@@ -1,31 +1,36 @@
-/* eslint-disable @stylistic/jsx-one-expression-per-line */
+import classNames from 'classnames/bind';
+import { dayJS } from '@/app/services/dayjs';
 import micIcon from '@/shared/assets/icons/mic.svg';
 import noteIcon from '@/shared/assets/icons/note-icon.svg';
 import Image from '@/shared/ui/image';
 
 import styles from './date-lang.module.scss';
 
+const cx = classNames.bind(styles);
+
 interface DateLangProps {
   startDate: string;
   mode: string;
   language: string[];
-  type?: string;
+  withMargin?: boolean;
```

Comment:
The component should not optionally change the layout. If it is necessary to add a margin, we will do it in the entity/widget

Reply:
It was used with margins on course pages, and this margin breaks layout if reuse this component in course cards. Should i wrap it with container with margins in cards?